2.Portfolio/Portofilo/ViewController.swift

```diff
@@ -8,9 +8,17 @@
 
 import UIKit
 
-class ViewController: UIViewController,UIScrollViewDelegate {
+class ViewController: UIViewController, UIScrollViewDelegate {
     @IBOutlet var heroimg: UIImageView!
-    @IBOutlet var scrollview: UIScrollView!
+    @IBOutlet var scrollview: UIScrollView! {
+        didSet {
```

Comment:
I think it is better to separate property setting from viewDidLoad function. Because viewDidLoad may very long, if you set other UI property.